script.js

```diff
@@ -1,7 +1,19 @@
 // PopUp Object
 
 const popObjects = [
-
+  {
+    title: 'Awesome books',
+    subtitle: 'Microverse',
+    experience: ['Front End Dev', '2023'],
+    image: './images/screenshots/MainPage.png',
+    image1: './images/screenshots/BooksList.png',
+    description1: 'An application that allows users to add or remove books from a list, not subscription is needed nor online conection.',
+    description: 'Awesome Books is a website that allows users to add/remove books from a list. It was achieved by using JavaScript objects and arrays. Dynamically modify the DOM and added basic events, using ES6 sintax, using modules into javascript code, and use of arrow functions.',
+    tech: ['html', 'css', 'javaScript', 'Bootstrap', 'ES6 syntax'],
+    seeLive: 'https://asadmisrani.github.io/AwesomeBooks/',
+    seeSource: 'https://github.com/asadmisrani/AwesomeBooks-ES6',
+    class: 'popCard1',
+  },
   {
```

Comment:
- [ ] I went further to investigate and realized in your object for the first card has the field for ```image1``` which is not present in all other objects. When I duplicated the ```image1``` in the second and third objects, the popup displayed the images as it is supposed to. 

[Screencast from 13-04-2023 07:25:08 ASUBUHI.webm](https://user-images.githubusercontent.com/86318284/231652893-0c997b1b-d817-4ac6-b4a9-ea67a869c3ac.webm)